src/setup.ts

```diff
@@ -9,4 +10,5 @@ export const setup = (config?: Config) => {
   if (config?.registerExpoDevMenuItem !== false) {
     registerExpoDevMenuItem();
   }
+  registerReactNativeDevMenuItem();
```

Comment:
on peut pas automatiser le bon call pour n'avoir q'une methode ScreenSizer.init() de publique ? Ex:

```
let isExpoInstalled = false;
let isExpoDevClientInstalled = false;
 try {
  isExpoInstalled = require('expo-dev-client')!!;
} catch (err) {
  isExpoInstalled = false;
}
try {
  isExpoDevClientInstalled = require('expo-dev-client')!!;
} catch(err) {
isExpoDevClientInstalled = false
}


if (isExpoInstalled) {
 if(isExpoDevClientInstalled) { registerExpoDevMenuItem } else { throw Error("add expo-dev-menu")}
} else {
registerReactNativeDevMenuItem
}
```

Reply:
ça serait cool oui :) Et en termes d'orga du code, je suggère de mettre le check à l'intérieur de `registerExpoDevMenuItem` (que ça ne fasse rien si pas installé)

Reply:
Je peux enlever le boolean de config oui il sert pas à grand chose, par contre dans le cas d'expo go on veut pouvoir utiliser la lib sans expo-dev-client (vu que l'ajout au dev menu dans le cas d'expo cas ne fonctionne pas), donc je serais plutôt partant pour ne pas faire de check sur expo et juste de faire un log si expo-dev-client n'est pas installé

Reply:
J'ai retravaillé le script de cette manière du coup, pour l'ajout du raccourci au expo dev menu :

|  | `expo-dev-menu` missing | `expo-dev-menu` installed |
| --- | --- | --- |
| bare React Native | ✅ (log: `expo dev menu skipped`) | ❌ (warning: addToMenu failed) |
| expo go | ✅ (log: `expo dev menu skipped`) | ❌ (warning: addToMenu failed) |
| expo (custom build) | ✅ (log: `expo dev menu skipped` - fonctionne mais le user n'a pas toutes les fonctionnalités) | ✅ (added to expo dev menu) |